src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java

```diff
@@ -124,7 +124,7 @@ protected static Configuration getModuleConfig(Configuration masterConfig, Strin
         final Configuration result;
         final List<Configuration> configs = getModuleConfigs(masterConfig, moduleName);
         if (configs.size() == 1) {
-            result = configs.get(0);
```

Comment:
hm i had no changed, this is 21 and will therefore break. 

- org.openrewrite.java.migrate.UpgradeToJava17
- https://github.com/checkstyle/checkstyle/issues/18148